packages/calcite-components/src/components/shell/shell.stories.ts

```diff
@@ -360,7 +360,7 @@ darkModeRTL_TestOnly.parameters = { themes: modesDarkDefault };
 
 export const closedPanels = (): string =>
   html(`<calcite-shell content-behind>
-  <calcite-shell-panel slot="panel-start" display-mode="float">
+  <calcite-shell-panel slot="panel-start" display-mode="float-detach">
```

Comment:
@josercarcamo I think we should still keep `float` stories since that value isn't being removed. If we just change this then we lose coverage of its functionality. Can you add new screenshot tests to cover float-detach?

Reply:
I don't think we're on the same page: "float-detach" is taking the place of the old "float", and "float" is the new fuctionality.

Reply:
Ok, but in the story updates its only replacing `float` which mean `float` would no longer have any test coverage. Shouldn't it?